//package com.sincetimes.redwar.game.comnunicate.protocol;


message RseUpdateItem
{
	optional int32 ret = 1; //1：资源不足, 2:创建失败, 3:找不到殖民地,4：销毁失败
	optional int32 id = 2;
	optional int32 sid = 3;
}